BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+        "Default": {"bg": "#ffffff", "fg": "#000000"},
+        "Dark": {"bg": "#121212", "fg": "#ffffff"},
+        # Add all your other themes here
+    }
+    selected_theme = themes.get(theme, themes["Default"])
+    var1["bg"] = selected_theme["bg"]
+    var2["fg"] = selected_theme["fg"]
+
+
+def restart_application():
+    python = sys.executable
+    os.execl(python, python, *sys.argv)
```

Ruff (0.12.2)

Starting a process without a shell

(S606)